docs/channels.md

```diff
@@ -0,0 +1,103 @@
+| `/x/pins` | [channel-pins](https://github.com/tloncorp/landscape-apps/blob/hm/hackweek/desk/mar/channel/pins.hoon) | Get pinned channels. Returns (list [$nest](https://github.com/tloncorp/landscape-apps/blob/hm/hackweek/desk/sur/channel.hoon#L184)) | <code>.^((list nest:c) %gx /=channels=/pins/channel-pins)</code>
+| `/x/[kind]/[ship]/[name]/perm` | [channel-perm](https://github.com/tloncorp/landscape-apps/blob/hm/hackweek/desk/mar/channel/perm.hoon) | Get permissions for a channel. Returns [$perm](https://github.com/tloncorp/landscape-apps/blob/hm/hackweek/desk/sur/channel.hoon#L227) | <code>.^(perm:c %gx /=channels=/chat/~sampel-palnet/mychat/perm/channel-perm)</code>
+| `/x/[kind]/[ship]/[name]/search/text/[skip=@]/[count=@]/[nedl=@]` | [channel-scan](https://github.com/tloncorp/landscape-apps/blob/hm/hackweek/desk/mar/channel/scan.hoon) | Search for text. Returns [$scan](https://github.com/tloncorp/landscape-apps/blob/hm/hackweek/desk/sur/channel.hoon#L195) | <code>.^(scan:c %gx /=channels=/chat/~sampel-palnet/mychat/search/text/0/10/'foo bar')</code>
+| `/x/[kind]/[ship]/[name]/search/mention/[skip=@]/[count=@]/[nedl=@]` | [channel-scan](https://github.com/tloncorp/landscape-apps/blob/hm/hackweek/desk/mar/channel/scan.hoon) | Search for a mention. Returns [$scan](https://github.com/tloncorp/landscape-apps/blob/hm/hackweek/desk/sur/channel.hoon#L195) | <code>.^(scan:c %gx /=channels=/chat/~sampel-palnet/mychat/search/text/0/10/~mister-hidden)</code>
+| `/x/[kind]/[ship]/[name]/posts/...` | [channel-posts](https://github.com/tloncorp/landscape-apps/blob/hm/hackweek/desk/mar/channel/posts.hoon) | Get posts from a channel. This can be called with a large variety of arguments, found [here](https://github.com/tloncorp/landscape-apps/blob/hm/hackweek/desk/app/channels.hoon#L1044). Returns [paged-posts](https://github.com/tloncorp/landscape-apps/blob/hm/hackweek/desk/sur/channel.hoon#L423) | <code>.^(scan:c %gx /=channels=/chat/~sampel-palnet/mychat/posts/newest/10/post</code>
+| `/u/[kind]/[ship]/[name]` | loob | Check the existence of a channel | <code>.^(? %gu /=channels=/chat/~sampel-palnet/mychat/loob)</code>
+
+## Pokes
+All pokes here have the [channel-action](https://github.com/tloncorp/landscape-apps/blob/hm/hackweek/desk/mar/channel/action.hoon) mark.
+
+### Channels
+| Description | Example
+|-------------|--------
+| Create a channel. This also updates the metadata in the `%groups` agent. See [$create-channel](https://github.com/tloncorp/landscape-apps/blob/hm/hackweek/desk/sur/channel.hoon#L245) | <pre>=create-channel :\*<br>  %chat<br>  %mychat<br>  [our %mygroup]<br>  title='My chat channel'<br>  description='Chat about things'<br>  readers=\~<br>  writers=\~<br>==<br><br>:channels &channel-action [%create create-channel]</pre>
```

Comment:
Wondering if these would benefit from having a first column that super briefly describes the shape or tag of the action.

Maybe like `[%create *]`, `[%channel * %unwatch *]`, `[%channel * %post %reply * %del-react *]`,  
or even terser like `%create`, `%unwatch`, `%del-react` respectively, and then just mentioning the `[%channel *]` case and such, as pointers to the following section.

Unsure if that would be better or not, but currently feels like there's some indexing/skimmability stuff missing here. The descriptions themselves are a bit too verbose for that.